src/CodeStyle/Core/Analyzers/xlf/CodeStyleResources.zh-Hant.xlf

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-HANT" original="../CodeStyleResources.resx">
```

Comment:
@cristianosuzuki77 I notice many of these changes are just a change in casing of the target-language. Will the be consistent going forward?

Reply:
@JoeRobich yes, I'll ensure it.

Reply:
@cristianosuzuki77 Sounds good. Thanks!